examples/Fine-tuned_classification.ipynb

```diff
@@ -581,159 +516,158 @@
        "  </tbody>\n",
        "</table>\n",
        "</div>"
+      ],
+      "text/plain": [
+       "                                              prompt completion\n",
+       "0  From: gld@cunixb.cc.columbia.edu (Gary L Dare)...     hockey\n",
+       "1  From: smorris@venus.lerc.nasa.gov (Ron Morris ...     hockey\n",
+       "2  From: golchowy@alchemy.chem.utoronto.ca (Geral...     hockey\n",
+       "3  From: krattige@hpcc01.corp.hp.com (Kim Krattig...   baseball\n",
+       "4  From: warped@cs.montana.edu (Doug Dolven)\\nSub...   baseball"
       ]
      },
+     "execution_count": 55,
      "metadata": {},
-     "execution_count": 13
+     "output_type": "execute_result"
     }
    ],
-   "metadata": {}
+   "source": [
+    "test = pd.read_json('sport2_prepared_valid.jsonl', lines=True)\n",
+    "test.head()"
+   ]
   },
   {
    "cell_type": "markdown",
+   "metadata": {},
    "source": [
     "We need to use the same separator following the prompt which we used during fine-tuning. In this case it is `\\n\\n###\\n\\n`. Since we're concerned with classification, we want the temperature to be as low as possible, and we only require one token completion to determine the prediction of the model."
-   ],
-   "metadata": {}
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": 14,
-   "source": [
-    "ft_model = 'ada:ft-openai-2021-07-30-12-26-20'\n",
-    "res = openai.Completion.create(model=ft_model, prompt=test['prompt'][0] + '\\n\\n###\\n\\n', max_tokens=1, temperature=0)\n",
-    "res['choices'][0]['text']\n"
-   ],
+   "execution_count": 69,
+   "metadata": {},
    "outputs": [
     {
-     "output_type": "execute_result",
      "data": {
       "text/plain": [
        "' hockey'"
       ]
      },
+     "execution_count": 69,
      "metadata": {},
-     "execution_count": 14
+     "output_type": "execute_result"
     }
    ],
-   "metadata": {}
+   "source": [
+    "ft_model = fine_tune_results.fine_tuned_model\n",
+    "# note that thsi calls the legacy completions api\n",
+    "res = client.completions.create(model=ft_model, prompt=test['prompt'][0] + '\\n\\n###\\n\\n', max_tokens=1, temperature=0)\n",
```

Comment:
It should be chat. completion. I guess many code blocks still rely on old API. it needs to be updated and the file should be tested. or else we should revert to the original and add a deprecation of the API notice on the top.

Reply:
I did test this and it works - it relies on the legacy completions api which has no ".chat." and the prompt= param. I'll push a fix for my typo ("thsi").

I don't think it makes too much sense to have this file for much longer, but it does show you how you can use babbage-002 (instead of the deprecated ada) for classification.

Reply:
okay then.